options.go

```diff
@@ -177,20 +177,13 @@ func (o *Options) Validate() error {
 			o.CookieExpire.String()))
 	}
 
-	// Todo - Read up on Go syntax and clean this up
 	// Confirm the provider type supports sending user roles
 	if o.PassRolesHeader {
-		type RoleProvider interface {
-			GetUserRoles() string
-		}
-		if rp, ok := o.provider.(RoleProvider); ok{
-			rp.GetUserRoles();
-		} else {
-			msgs = append(msgs, "Provider '" + o.provider.Data().ProviderName + "' does not support sending a roles header.")
+		if _, ok := o.provider.(providers.RoleProvider); !ok {
```

Comment:
The use of _ when you need a place filled but aren't going to use the variable is the notable thing here, I think, @tlots.